index.js

```diff
@@ -517,6 +655,11 @@ class SuperExpressive {
       case 'namedBackreference': return `\\k<${el.metadata}>`;
       case 'backreference': return `\\${el.metadata}`;
 
+      case 'subexpression': {
```

Comment:
Might be better to mark `t.subexpression` with `quantifierRequiresGroup`, and let any quantifiers deal with adding the group.